### PR TITLE
WebView Sample java

### PR DESCRIPTION
**Pull request by rkuziv**

### CODE REVIEW

File: ketch-android/Android Ketch Pref Center using Java/app/src/main/java/com/ketch/sample/webview/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class MainActivity extends AppCompatActivity {
+    private static final String TAG = MainActivity.class.getSimpleName();
+
+    private static final String ORG_CODE = "transcenda";
+    private static final String PROPERTY = "website_smart_tag";
```
Comment (OnyxPrime) [inline ketch-android/Android Ketch Pref Center using Java/app/src/main/java/com/ketch/sample/webview/MainActivity.java:26]: Please update with placeholder values

**Review by OnyxPrime — CHANGES_REQUESTED** [review]

File: ketch-android/Android Ketch Pref Center using Java/app/src/main/java/com/ketch/sample/webview/MainActivity.java (hunk 1 above)
Comment (rkuziv) [reply ketch-android/Android Ketch Pref Center using Java/app/src/main/java/com/ketch/sample/webview/MainActivity.java:26]: ok

File: ketch-android/Android Ketch Pref Center using Java/README.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+Advertising ID code `ADVERTISING_ID_KEY` is available on the [Identity Spaces](https://app.ketch.com/settings/identity-spaces)
+
+The preferences screen is triggered on the button click in this example, but it could also be
+triggered automatically on application start.
+
+```java
+public class MainActivity extends AppCompatActivity {
+    
+    private static final String ORG_CODE = "transcenda";
+    private static final String PROPERTY = "website_smart_tag";
```
Comment (OnyxPrime) [inline ketch-android/Android Ketch Pref Center using Java/README.md:129]: Please update these to placeholder values.

**Review by OnyxPrime — CHANGES_REQUESTED** [review]

File: ketch-android/Android Ketch Pref Center using Java/README.md (hunk 2 above)
Comment (rkuziv) [reply ketch-android/Android Ketch Pref Center using Java/README.md:129]: oh, I will fix

**Review by OnyxPrime — APPROVED** [review]
LGTM. Thank you!

**Review by webdevbyjoss — APPROVED** [review]
LGTM

**Pull request merged**